yt/data_objects/tests/test_data_containers.py

```diff
@@ -61,9 +61,10 @@ def test_yt_data_container(self):
 
     def test_write_out(self):
         filename = "sphere.txt"
-        ds = fake_random_ds(16)
+        ds = fake_random_ds(16, particles=10)
```

Comment:
curious: why did you add this kwarg ?

Reply:
It is to check that writing out the dataset to a file works even if the dataset contains particles and mesh data. This is the complement of `test_invalid_write_out` which tests that writing the dataset *doesn't* work if you try to dump both mesh and particle data at the same time.

Reply:
excellent, thanks